apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -221,6 +226,130 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    /**
+     * Retrieves scopes with the details from the API.
+     *
+     */
+    const getScopes = () => {
+
+        getOIDCScopesList()
+            .then((response: OIDCScopesListInterface[]) => {
+                setScopes(response);
+            })
+            .catch((error) => {
+                if (error.response && error.response.data && error.response.data.description) {
+                    dispatch(
+                        addAlert({
+                            description: error.response.data.description,
+                            level: AlertLevels.ERROR,
+                            // todo: check whether you need to change the error message
+                            message: t(
+                                "console:manage.features.oidcScopes.notifications."
+                                + "fetchOIDCScope.error.message"
+                            )
+                        })
+                    );
+
+                    return;
+                }
+
+                dispatch(
+                    addAlert({
+                        // todo: check whether you need to change the error message
+                        description: t(
+                            "console:manage.features.oidcScopes.notifications.fetchOIDCScope" +
+                            ".genericError.description"
+                        ),
+                        level: AlertLevels.ERROR,
+                        // todo: check whether you need to change the error message
+                        message: t(
+                            "console:manage.features.oidcScopes.notifications.fetchOIDCScope.genericError."
+                            + "message"
+                        )
+                    })
+                );
+            });
+    };
+
+    /**
+     * Check whether claim is mandatory or not
+     *
+     * @param uri - Claim URI to be checked.
+     * @returns If initially requested as mandatory.
+     */
+    const checkInitialRequestMandatory = (uri: string): boolean => {
+        const externalClaim = externalClaims.find((claim) => claim.claimURI==uri);
+
+        if (externalClaim){
+            const requestURI = claimConfigurations.requestedClaims.find(
+                (requestClaims) => (requestClaims?.claim?.uri === externalClaim.mappedLocalClaimURI))?.mandatory;
+
+            if (requestURI!= undefined) {
```

Comment:
requestURI!= change to requestURI !=
```suggestion
            if (requestURI != undefined) {
```